vmware_aria_operations_integration_sdk/mp_build.py

```diff
@@ -147,15 +146,22 @@ def get_docker_registry(
     **kwargs: Any,
 ) -> str:
     docker_registry = get_config_value("docker_registry", config_file=config_file)
-    default_registry_value = f"harbor-repo.vmware.com/vmware_aria_operations_integration_sdk_mps/{adapter_kind_key.lower()}"
+    default_registry_value = get_config_value("default_container_registry")
 
     original_value = docker_registry
     if docker_registry is None and docker_registry_arg is None:
         print(
             "mp-build needs to configure a container registry to store the adapter container image.",
             "class:information",
         )
-        docker_registry = registry_prompt(default=default_registry_value)
+        if default_registry_value is not None:
```

Comment:
Do we need to verify that this value is a valid domain? If we are setting a default, we should make sure the default is valid (This is why we wanted to remove the VMware registry)